stacks_queues/stack.py

```diff
@@ -12,27 +13,33 @@ def push(self, element):
         """ Adds an element to the top of the Stack.
             Returns None
         """
-        pass
+        self.store.add_last(element)        
 
     def pop(self):
         """ Removes an element from the top
             Of the Stack
             Raises a StackEmptyException if
             The Stack is empty.
-            returns None
         """
-        pass
+        if self.store is None:
+            raise StackEmptyException('Stack is empty')
+
+        return self.store.remove_last() 
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        return self.store.empty()
 
     def __str__(self):
         """ Returns the Stack in String form like:
             [3, 4, 7]
             Starting with the top of the Stack and
             ending with the bottom of the Stack.
         """
-        pass
+        result = []
+        while not self.empty():
+            result.append(self.pop())
+
+        return f"{result}"
```

Comment:
This will destroy the contents of the stack if we try to print it, since `self.pop()` takes things out of the list. The LinkedList class itself has a `__str__` method we can use, but we need to store items at the front for it to print the contents in the desired order.